integration-test/common/src/test/java/org/apache/uniffle/test/AssignmentWithTagsTest.java

```diff
@@ -78,65 +80,45 @@ private static List<Integer> findAvailablePorts(int num) throws IOException {
     return ports;
   }
 
-  private static void createAndStartShuffleServerWithTags(Set<String> tags, File tmpDir)
-      throws Exception {
-    ShuffleServerConf shuffleServerConf = getShuffleServerConf(ServerType.GRPC);
+  private static void prepareShuffleServerConf(int subDirIndex, Set<String> tags, File tmpDir) {
+    ShuffleServerConf shuffleServerConf =
+        shuffleServerConfWithoutPort(subDirIndex, tmpDir, ServerType.GRPC);
```

Comment:
with this pr, some test helper methods will become unused. Could you please clean them up together in current one? such as `AssignmentWithTagsTest#findAvailablePorts`